src/utils.ts

```diff
@@ -294,7 +294,11 @@ function getProjectReferenceForFile(filePath: string, instance: TSInstance) {
         .getProjectReferences()!
         .find(
           ref =>
-            (ref && ref.commandLine.fileNames.indexOf(filePath) > -1) || false
+            (ref &&
+              ref.commandLine.fileNames.some(
+                file => path.normalize(file) === filePath
```

Comment:
Running `typescript.normalizePath` on `filePath` would be more straightforward, but that function seems to be private.